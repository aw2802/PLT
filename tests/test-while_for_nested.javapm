public class TestWhileForNested {
  public void main(){
    int i = 0;
    int j;
    while(i < 2){
      for(j = 0; j < 2; j = j + 1){
      	println("Inside first test");
      }  
      i = i + 1;
    }
    println();
    
    for(i = 0; i < 2; i = i + 1){
      j = 0;
      while(j < 2){
        println("Inside second test");
	j = j + 1;
      }
    }
  } 
}
