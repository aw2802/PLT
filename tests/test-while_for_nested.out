Inside first test
Inside first test
Inside first test
Inside first test

Inside second test
Inside second test
Inside second test
Inside second test
